src/main/java/octo/waddle/hellodropwizard/resources/HelloWorldResource.java

```diff
@@ -0,0 +1,33 @@
+package octo.waddle.hellodropwizard.resources;
+
+import octo.waddle.hellodropwizard.api.Message;
+import com.codahale.metrics.annotation.Timed;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.Optional;
+
+@Path("/hello")
+@Produces(MediaType.APPLICATION_JSON)
+public class HelloWorldResource {
+
+    private final String template;
+    private final String defaultName;
+    private final AtomicLong counter;
+
+    public HelloWorldResource(String template, String name) {
+        this.template = template;
+        this.defaultName = name;
+        this.counter = new AtomicLong();
+    }
+
+    @GET
+    @Timed
+    public Message sayHello(@QueryParam("name") Optional<String> name) {
```

Comment:
I nie używaj optionali jako argumenty metod
To że można nie oznacza że się powinno :)
https://stackoverflow.com/questions/31922866/why-should-java-8s-optional-not-be-used-in-arguments